Icon/style.css

```diff
@@ -1,13 +1,108 @@
 @import '../variables.css';
 
+.curious-blue {
```

Comment:
I think size would (or should) all be handled by the `Icon` component. I'd expect this to be a pass through.